Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            const int maxEntries = 3;
```

Comment:
This variable let's tre reader think that there will be 3 entries in the end. This is unfortunate because there are going to be only 2.

To fix this do the follwing:
Replace the 3 with a 2 and replace `i < maxEntries` with `i <= maxEntries`.
This won't change your program in any way but make it much more clearer (and correct from logical perspective).